package.json

```diff
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "start": "nf start",
+    "start:dev": "nf start -f Profile_dev",
```

Comment:
Typo here, should be `Procfile`, not `Profile`.  :P Also, please rename file to Procfile.dev